Dockerfile

```diff
@@ -51,44 +52,61 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 	libtool \
 	libunistring-dev \
 	lzip \
-	mercurial \
+	man-db \
+	manpages-dev \
+	mlocate \
 	moreutils \
-	ninja-build \
+	nasm \
 	patchelf \
 	php \
 	pkg-config \
-	python2 \
-	python3 \
-	python3-distutils \
 	rename \
+	ripgrep \
 	rsync \
 	ruby-mustache \
 	scons \
 	subversion \
 	sudo \
 	swig \
 	texinfo \
+	time \
+	tree \
 	unzip \
 	xmlto \
+	yasm \
 	zip \
-	zlib1g-dev && \
-	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
-	adduser --disabled-password --gecos '' user && \
-	adduser user sudo && \
-	echo "%user ALL=(ALL:ALL) NOPASSWD: ALL" | sudo tee /etc/sudoers.d/users
+	zlib1g-dev
+
+# Python based apps
+RUN apt-get update && apt-get install --no-install-recommends -y \
+	httpie \
+	mercurial \
+	meson \
+	ninja-build \
+	python3 \
+	python3-distutils \
+	python3-mako \
+	python3-pip \
+	python3-virtualenv \
+	python3-yaml
 
-# Install setuptools, wheel and pip for Python2
-RUN wget https://bootstrap.pypa.io/pip/2.7/get-pip.py -O - | python2
-# Install virtualenv and httpie for Python2
-# Use pip2 as default pip -> python3
-RUN pip2 install virtualenv httpie
+# Install hg github tool
+RUN curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg && \
+	echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null && \
+	apt update && \
+	apt install gh
```

Comment:
as commented, gh is installable in debian-bookworm without additional prerequisites...

Reply:
Thnx, now fixed and using default debian repository.

> But why should we have github command line tool?

I figured, when things goes sideways `gh` can be quite handy to play with your repository.  Totally optionnal, agreed, but as our project entirely sits under github, and its nice to have it around once in a while, it doesn't cost too much, further as now part of default debian repository.

I must admit, that I didn't had to use it in a while as I have a strong preference on `git`.  Although there may be something to do with it for future download options ... Unless you disagree I'd keep it included for now on?